rita/src/rita_client/rita_loop/mod.rs

```diff
@@ -42,6 +42,7 @@ impl Handler<Tick> for RitaLoop {
             ExitManager::from_registry()
                 .send(Tick {})
                 .into_actor(self)
+                .timeout(Duration::from_secs(5), MailboxError::Timeout)
```

Comment:
This timeout should either be configurable or computed as e.g. 150% of the average time the Tick takes , the 5 seconds are as arbitrary as it gets, half a radix we daily, derived from the total finger count humans have.

Reply:
I tried an arbitrary value to test, but this plain doesn't work, I'll go looking at the locks.